app/views/about/more.json.rabl

```diff
@@ -0,0 +1,10 @@
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
```

Comment:
I think there's a `site_title` method available for this, and instance presented has lots of the others